SimUI.py

```diff
@@ -136,18 +142,9 @@ def _webhandle_pause_POST(self):
     #=== web server admin
     
     def _bottle_try_running_forever(self,*args,**kwargs):
-        RETRY_PERIOD = 3
+
         while True:
             try:
                 args[0](**kwargs) # blocking
-            except Exception as err:
```

Comment:
why did you remove this? this is useful if the TCP port is occupied. replace the print statement by log.error or log.critical